### PR TITLE
User document view

### PR DESCRIPTION
**Pull request by nimish-keel**

### CODE REVIEW

File: dev/js/pages/UserDashboardView/DocumentView/UploadedDocument/style.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    font-size: 16px;
+    padding: 20px 20px;
+    line-height: 1.5rem;
+    font-family: 'Poppins';
+  }
+  .date {
+    justify-content: flex-start;
+    padding-left: 25px;
+    font-size: 10px;
+    font-family: Poppins;
```
Comment (Pankajkeel) [inline dev/js/pages/UserDashboardView/DocumentView/UploadedDocument/style.js:62]: dont use font family directly , plz use constant for this, Check that in your whole code & change it.

File: dev/js/pages/UserDashboardView/DocumentView/index.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
+import React, { useState, useEffect } from 'react';
+import Header from '@components/Header';
+import UploadedDocument from './UploadedDocument/index';
+import NotificationWidget from '@components/NotificationWidget';
+import ProfileWidget from '@components/ProfileWidget';
+import {
+  getUserDocuments,
+  uploadUserDocument,
+} from '../../../actions/index.js';
```
Comment (Pankajkeel) [inline dev/js/pages/UserDashboardView/DocumentView/index.js:9]: use @actions alias path

File: dev/js/pages/UserDashboardView/DocumentView/index.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+            <FileUpload
+              fileUploadModalClosed={fileuploadclosed}
+              uploadFile={fileUploadDone}
+            />
+          )}
+        </div>
+        <div className="uploadedDocsWrapper">
+          {userAddedDocs
+            .filter((val) => {
+              if (searchDoc === '') {
```
Comment (Pankajkeel) [inline dev/js/pages/UserDashboardView/DocumentView/index.js:130]: if else both condition return same data so why use if else ?

File: dev/js/pages/UserDashboardView/DocumentView/index.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                return val;
+              } else if (
+                val.doc_type.toLowerCase().includes(searchDoc.toLowerCase())
+              ) {
+                return val;
+              }
+            })
+            .map((doc) => {
+              if (documentOwner === '') {
```
Comment (Pankajkeel) [inline dev/js/pages/UserDashboardView/DocumentView/index.js:139]: both if condition return same data either merge it or remove duplicate if condition.

File: dev/js/pages/UserDashboardView/DocumentView/style.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+  }
+  .uploadSection {
+    display: flex;
+    justify-content: space-around;
+    height: 40px;
+  }
+  .uploadText {
+    font-size: 25px;
+    line-height: 2.5rem;
+    font-family: Poppins;
```
Comment (Pankajkeel) [inline dev/js/pages/UserDashboardView/DocumentView/style.js:25]: change font to constant

File: dev/js/pages/UserDashboardView/DocumentView/style.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    line-height: 27px;
+  }
+  .uploadButton {
+    border-radius: 15px;
+    margin-left: 5%;
+    width: 250px;
+    background: #363b64;
+    cursor: pointer;
+    color: #ffff;
+    font-family: Poppins;
```
Comment (Pankajkeel) [inline dev/js/pages/UserDashboardView/DocumentView/style.js:62]: change font to constant

File: dev/js/reducers/commons/document.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+const defaultState = {
+  userDocuments: [],
+  docTypes: [],
+};
+
+export default function (state = defaultState, action) {
+  switch (action.type) {
+    case GET_USER_DOCUMENTS: {
+      let newState = { ...state };
```
Comment (Pankajkeel) [inline dev/js/reducers/commons/document.js:17]: bro u didn't do anything in these reducers, u just return the same state ,so why dispatch action, if there is no need then remove the dispatch from action.

File: dev/js/reducers/commons/document.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+      return newState;
+    }
+    case GET_DOC_TYPES: {
+      let newState = { ...state };
+      return newState;
```
Comment (Pankajkeel) [inline dev/js/reducers/commons/document.js:22]: bro u didn't do anything in these reducers, u just return the same state ,so why dispatch action, if there is no need then remove the dispatch from action.

File: dev/js/pages/UserDashboardView/DocumentView/index.js (hunk 3 above)
Comment (nimish-keel) [reply dev/js/pages/UserDashboardView/DocumentView/index.js:130]: @Pankajkeel In both conditions the parameter val is changing based on the user input, thats why even thought the return is val but val is changing while user is typing.

File: dev/js/pages/UserDashboardView/DocumentView/UploadedDocument/style.js (hunk 1 above)
Comment (nimish-keel) [reply dev/js/pages/UserDashboardView/DocumentView/UploadedDocument/style.js]: Corrected

File: dev/js/pages/UserDashboardView/DocumentView/index.js (hunk 2 above)
Comment (nimish-keel) [reply dev/js/pages/UserDashboardView/DocumentView/index.js]: Done

File: dev/js/pages/UserDashboardView/DocumentView/index.js (hunk 4 above)
Comment (nimish-keel) [reply dev/js/pages/UserDashboardView/DocumentView/index.js]: Corrected

File: dev/js/pages/UserDashboardView/DocumentView/style.js (hunk 5 above)
Comment (nimish-keel) [reply dev/js/pages/UserDashboardView/DocumentView/style.js]: Done

File: dev/js/pages/UserDashboardView/DocumentView/style.js (hunk 6 above)
Comment (nimish-keel) [reply dev/js/pages/UserDashboardView/DocumentView/style.js]: Done

File: dev/js/reducers/commons/document.js (hunk 7 above)
Comment (nimish-keel) [reply dev/js/reducers/commons/document.js:11]: Corrected

Comment (Pankajkeel) [reply dev/js/reducers/commons/document.js:11]: bro you don't need to add user Document in redux store, as you are not using it right
u are using callback function to render data

**Review by Pankajkeel — APPROVED** [review]

**Pull request closed without merging**